pkg/policy/cell/policy_importer.go

```diff
@@ -311,7 +311,7 @@ func (i *policyImporter) processUpdates(ctx context.Context, updates []*policyty
 
 	// All policy updates have been applied; regenerate all affected endpoints.
 	// Unaffected endpoints can merely have their policy revision set.
-	i.log.Info("Policy repository updates complete, triggering endpoint updates",
+	i.log.Debug("Policy repository updates complete, triggering endpoint updates",
```

Comment:
I would possibly keep this log as info and add number of updates processed in this batch along with startRevision as logfields. We don't expect very high churn in policy objects so it should be ok.